pom.xml

```diff
@@ -98,7 +98,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>24.1.0</version>
+        <version>${cloud-libraries-bom.version}</version>
```

Comment:
@Steven-M-Brown this one

Reply:
The previous had a mismatch with the rest of the libraries. As a result, that caused a ton of warnings.